### ISSUE TITLE
Compiling to assembly book of your will be only cover disassembling typescript/js code.

### ISSUE THREAD

**Issue by coderj001** [keleshev/compiling-to-assembly-from-scratch#1]

**Comment by keleshev** [comment]
Hi @coderj001! I'm not sure I follow. Can you rephrase the issue or add some detail?

**Comment by coderj001** [comment]
The book "Compiling to assembly from scratch" I guess you are the author right. I just like to know will it will cover the assembly part or not. Because I have little to no idea about assembly.

**Comment by keleshev** [comment]
The book only assumes that you know how to program in a high-level language like JavaScript or Python. The book's chapter *Introduction to ARM Assembly Programming* teaches you enough assembly programming to complete the compiler.

By the way, the beginning of this chapter is available in the book's excerpt/preview: https://keleshev.com/compiling-to-assembly-from-scratch/excerpt-compiling-to-assembly-from-scratch.pdf. Check it out and see if it makes sense to you! 

Let me know if you have more questions!

**Comment by coderj001** [comment]
Thank you for explaining.
